### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cfl0ws**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +20,25 @@ This Gitbook serves as a place to capture the discussions as they evolve and for
...
-* What - what is governance, what areas, actions and policies does it cover
 * Who - who participates in governance, in which roles
+* What - what is governance, what areas, actions and policies does it cover
 * How - how do we implement
 
+## Principles
+
+TBD - Suggested in TG chat
+
+## Contributors
```
Comment (michaelh-laine) [inline README.md:36]: Github automatically displays this info (https://github.com/laine-sa/solana-governance-think-tank/graphs/contributors) wouldn't it be simpler than manually updating the README over time?

**Review by michaelh-laine — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cfl0ws) [reply README.md:36]: I think it would be nice to have a list in the Gitbook as my sense is many people who review the Gitbook won't look at the source.

Comment (michaelh-laine) [reply README.md:36]: Yeah but then just link to the Github page maybe?

Comment (cfl0ws) [reply README.md:36]: We could but I don't see the harm in listing contributors directly in the Gitbook and linking to the contributors page. I'll add the link and if you'd prefer to remove the list from the Gitbook, please go ahead and do that.

Comment (michaelh-laine) [reply README.md:36]: Thanks let's leave this as is for now, like I initially mentioned my concern is just maintenance if the contributor list grows

**Pull request merged**